captum/attr/_core/integrated_gradients.py

```diff
@@ -16,6 +17,7 @@
     _expand_target,
 )
 from .._utils.attribution import GradientAttribution
+from .._utils.typing import TensorOrTuple
```

Comment:
nit: let's rename it to `TensorOrTupleOfTensors` - then it is very explicit and users don't have to think what is in the tuple ?

Reply:
Sounds good, yeah, I agree that would be better naming 👍!